extension/duckdb/src/duckdb_scan.cpp

```diff
@@ -18,7 +18,7 @@ void getDuckDBVectorConversionFunc(PhysicalTypeID physicalTypeID,
 DuckDBScanBindData::DuckDBScanBindData(std::string query,
     std::vector<common::LogicalType> columnTypes, std::vector<std::string> columnNames,
     const DuckDBConnector& connector)
-    : TableFuncBindData{std::move(columnTypes), std::move(columnNames)}, query{std::move(query)},
+    : TableFuncBindData{std::move(columnTypes), std::move(columnNames)}, 0, query{std::move(query)},
```

Comment:
As a convention, we always annotate constant `0 /* numWarningColumns */`. Apply this across this PR

Reply:
updatede